### PR TITLE
APP-4023: Add wrapper for RobotService.restartModule

### PR DESCRIPTION
**Pull request by michaellee1019**

### CODE REVIEW

File: src/robot/client.ts [hunk 1]
```diff
@@ -813,4 +813,23 @@ export class RobotClient extends EventDispatcher implements Robot {
...
+      request.setModuleId(moduleId);
+    }
+    if (moduleName !== undefined) {
+      request.setModuleName(moduleName);
+    }
+    const response = await promisify<
+      proto.RestartModuleRequest,
+      proto.RestartModuleResponse
+    >(robotService.restartModule.bind(robotService), request);
+    return response.toObject();
```
Comment (njooma) [inline src/robot/client.ts:833]: Probably don't need to actually return anything since the return type doesn't include any data

**Review by njooma — CHANGES_REQUESTED** [review]

File: src/robot/client.ts (hunk 1 above)
Comment (michaellee1019) [reply src/robot/client.ts:833]: Yeah makes sense. If we add fields to the response in the future, then we can return it.

**Review by stuqdog — CHANGES_REQUESTED** [review]
Can we add a stub for the `restartModule` method to `robot/robot.ts` with a docstring that explains the method a bit (what it does, if/when it raises, behavior when neither param is provided, stuff like that)?

**Comment by michaellee1019** [comment]
> Can we add a stub for the `restartModule` method to `robot/robot.ts`

Ah, didn't realize this existed. Will do!

**Review by stuqdog — APPROVED** [review]

**Review by njooma — APPROVED** [review]

**Pull request merged**